app/src/main/res/xml/player_settings.xml

```diff
@@ -133,6 +133,12 @@
             android:title="@string/queue_insert_related_videos"
             app:key="queue_insert_related_videos" />
 
+        <SwitchPreferenceCompat
+            android:defaultValue="true"
```

Comment:
This should default to false, since most users expect autoplay to impact the normal queue and PLAYLISTS. This way, if users disable autoplay via the player UI, it might appear like a bug that playlists still autoplay to new users.